sormas-backend/src/main/java/de/symeda/sormas/backend/document/DocumentFacadeEjb.java

```diff
@@ -100,6 +111,27 @@ public DocumentDto saveDocument(@Valid DocumentDto dto, byte[] content) throws I
 
 		Document document = fillOrBuildEntity(dto, existingDocument,true);
 
+		String[] allowedFileExtensions = configFacade.getAllowedFileExtensions();
```

Comment:
I've wanted to test the document upload but NPE is thrown for DocumentFacade and the Document card is no longer appearing(even if the logged in user has the proper rights) for cases or events. I also tested on development and the Documents card is there so probably the changes from this PR should be debugged:

![image](https://user-images.githubusercontent.com/47103965/199284744-9f3f09ac-bc00-4100-b2ad-3f789c074aa8.png)


![ErrorForRazvan2](https://user-images.githubusercontent.com/47103965/199285054-372ab584-fa5b-46a1-8873-b77352e99d09.PNG)

I attach also the error log:
[Error_Log_9965.txt](https://github.com/hzi-braunschweig/SORMAS-Project/files/9912227/Error_Log_9965.txt)

Reply:
Tested it again locally, everything works for me, attached screenshots

![Screenshot 2022-11-02 at 14 31 43](https://user-images.githubusercontent.com/17514879/199491251-75d160d4-9542-4a93-aa75-52f6d99bdc34.png)
![Screenshot 2022-11-02 at 14 32 20](https://user-images.githubusercontent.com/17514879/199491255-9e15b05e-0f3e-47f8-86a6-7d79852aeb1f.png)
![Screenshot 2022-11-02 at 14 32 31](https://user-images.githubusercontent.com/17514879/199491258-f58d7439-de06-44fe-8200-4ce55b671bb8.png)
![Screenshot 2022-11-02 at 14 32 45](https://user-images.githubusercontent.com/17514879/199491263-55f27e89-7fa6-47ed-adb4-d5de295d607a.png)